internal/value/value.go

```diff
@@ -1027,6 +1327,10 @@ type optionalValue struct {
 	value     Value
 }
 
+func (v *optionalValue) CastTo(dst interface{}) error {
+	return v.value.CastTo(dst)
```

Comment:
I think about cast optional type must be check about dst is double pointer and check/save nil value if value is null